data/en/mbs_0158.json

```diff
@@ -26,7 +26,7 @@
                             "id": "use-of-information",
                             "content": [{
                                 "list": [
-                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
+                                    "Data should relate to all sites in England, Scotland, Wales.",
```

Comment:
In my surveys I made it `Data should relate to all sites in England, Scotland and Wales.`